src/style-config/colors.scss

```diff
@@ -4,6 +4,13 @@
 $white:                      #FFF !default;
 $black:                      #000 !default;
 
+// Text Colors
+$onyx:                       rgba(0,0,0,0.85) !default;
```

Comment:
Did these work on the demo site?

You may have to run the colors script to generate them

Reply:
So I checked and only base-colors show in demo site, so this is not a problem, if we did move these to base colors, the demo site would not work, therefore best to leave these here until you can fix the code that is rendering the base-colors.